.gitignore

```diff
@@ -13,3 +13,4 @@ pom.xml
 .repl
 .nrepl-port
 *.swp
+.DS_Store
```

Comment:
Adding that to my global `~/.gitignore` did not make git ignore it in this repo.